.buildkite/pipeline.yml

```diff
@@ -923,6 +953,19 @@ steps:
           slurm_gpus: 1
         soft_fail: true
 
+      - label: "GPU: gpu_aquaplanet_dyamond"
+        command:
+          - mkdir -p gpu_aquaplanet_dyamond
+          - > 
+            nsys profile --trace=nvtx,cuda --output=gpu_aquaplanet_dyamond/report
+            julia --color=yes --project=examples examples/hybrid/driver.jl
+            --config_file ${GPU_CONFIG_PATH}gpu_aquaplanet_dyamond.yml
+        artifact_paths: "gpu_aquaplanet_dyamond/*"
+        agents:
+          slurm_gpus: 1
+          slurm_time: 23:00:00
```

Comment:
same here, I think the ci requires simulation shorter than an hour.

Reply:
I removed the `slurm_time` entry. We can add additional GPUs if this is slower!